src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -232,6 +233,16 @@ class ReimbursementAccountPage extends React.Component {
                             error={error}
                         />
                     )}
+                    {currentStep === CONST.BANK_ACCOUNT.STEP.VALIDATION && (
+                        <ValidationStep
+                            achData={this.props.reimbursementAccount.achData}
+                            maxAttemptsReached={maxAttemptsReached}
+                            error={error}
+                        />
+                    )}
+                    {currentStep === CONST.BANK_ACCOUNT.STEP.EXISTING_OWNERS && (
+                        <ExistingOwnersStep achData={achData} />
```

Comment:
Steps have a pretty specific significance and are tied to the backend. If we just want to show an error inline here then we can do something similar to the lines above where we are setting the `errorComponent` if there's a problem we need to handle.